vsts/pipelines/templates/_platformBinariesTemplate.yml

```diff
@@ -19,6 +19,8 @@ steps:
 
 - task: ShellScript@2
   displayName: 'Building platform binaries'
+  env:
+    PRIVATESTORAGESASTOKEN: $(PrivateStorageSASToken)
```

Comment:
Just realized that this name is almost exactly the same as PRIVATE_STORAGE_SAS_TOKEN that we set to the value in our key vault (in _platformBinariesReleaseTemplate.yml). I would lean towards updating this to be the name that we discussed in our meeting:
```
    env:
      DOTNET_PRIVATE_STORAGE_ACCOUNT_ACCESS_TOKEN: $(DotnetPrivateStorageAccountAccessToken)
```

this would also align it with the variable that I'm using for the base images job